plugins/kinet/KiNetNode.cpp

```diff
@@ -125,8 +131,8 @@ bool KiNetNode::SendDMX(const IPV4Address &target_ip, const DmxBuffer &buffer) {
 bool KiNetNode::SendPortOut(const IPV4Address &target_ip,
                             const uint8_t port,
                             const DmxBuffer &buffer) {
-  static const uint8_t flags = 0;
-  static const uint8_t padding = 0;
+  static const uint8_t flags1 = 0; // Definitely flags of some sort, seems to be impacted by buffer size
```

Comment:
Could you be more specific how the buffer size impacts it?

Reply:
I'm not particularly sure, but I will make a key-value table so we can figure it out.

Reply:
See below, I'm no longer sure what this is, but it seems to be safe to leave at `0x00`.